Casbin.Persist.Adapter.EFCore/EFCoreAdapter.cs

```diff
@@ -94,6 +98,10 @@ public virtual async Task SavePolicyAsync(IPolicyStore store)
                 return;
             }
 
+            var existRule = PersistPolicies.ToList();
```

Comment:
It will select all rows to memory, plz use truncate here. https://github.com/casbin/gorm-adapter-ex/blob/474b4560365b406ea7cf008619a445fb7b5e24c7/adapter.go#L551

Reply:
it seems that efcore not officially support truncate，i think use raw sql query may cause some compatibility problem when using some database